src/index.html

```diff
@@ -11,6 +11,15 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
```

Comment:
Still not fixed from the previous review, this div is redundant here
```suggestion
```

Reply:
Please, check demo link again ([DEMO LINK](https://alyatulupova.github.io/layout_search-bar-airbnb/), because this changes were done. The last version:
<img width="1324" alt="Знімок екрана 2023-11-25 о 22 38 43" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/149272263/e5660c44-000f-4028-9705-a991cf9510da">

Reply:
but we still have this block in git, check you code in PR before re-request review and ask in fe_chat if you have some questions